style.css

```diff
@@ -0,0 +1,80 @@
+* {
```

Comment:
[OPTIONAL] Kindly don't forget to import the style you want to use on your website. You can check the example below:

```@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');```

Right now your website is not using the `Poppins` font instead `sans-serif` is being used since Poppins is not imported. Importing the font should fix your issues.